template/script.sh.erb

```diff
@@ -16,12 +16,11 @@ end
 
 # Setup environment.
 APP_ROOT="<%= app_root.to_s %>"
-CODE_SERVER_DATAROOT="$HOME/.local/share/bc_osc_codeserver"
+CODE_SERVER_DATAROOT="${CODE_SERVER_DATAROOT:=$HOME/.local/share/bc_osc_codeserver}"
```

Comment:
`:-` is appropriate here since we are setting the value of that expression to CODE_SERVER_DATAROOT